js/holmes.js

```diff
@@ -86,42 +86,45 @@
      * Options
      * @type {Object}
      */
-    holmes.prototype.options = options;
+    holmes.prototype.options = {
+      input: 'input[type=search]',
+      placeholder: '',
+      placeholderNode: null,
```

Comment:
It is for optimize V8 hidden classes.
As u know, adding properties in object make V8 create new hidden class.

Also if variables has same types, V8 can optimize it.